fuzzing/scripts/custom-build.sh

```diff
@@ -240,55 +240,58 @@ fi
 llvm_sanitizer=""
 
 # Address, Memory, and Thread Sanitizers are incompatibile.
+# Also, UndefinedBehavior should only be used by itself or with Address.
 # See clang/lib/Driver/SanitizerArgs.cpp#IncompatibleGroups
-print_q   "Add AddressSanitizer, MemorySanitizer, or ThreadSanitizer?"
+print_q   "Add AddressSanitizer, MemorySanitizer, ThreadSanitizer, UndefinedBehaviorSanitizer, or None?"
```

Comment:
Maybe put this into two lines to avoid overlong output.

Reply:
Factored out the "Sanitizer" bits to make this short enough. Might want to change this to list things out like the optimization levels, but that runs into issues with keeping the links around. Shortening seems fine for now.